### DISCUSSION TITLE
为什么md格式文件渲染的页面不显示?

### DISCUSSION THREAD

**Discussion by dong-lufei** [posva/unplugin-vue-router#492] [Questions and Help] [answered]
![屏幕截图 2024-08-24 183702](https://github.com/user-attachments/assets/315c6123-91b4-4421-8147-a657633b03a3)

**Comment by posva** [comment, marked as answer]
You need to support md with something like https://github.com/unplugin/unplugin-vue-markdown

**Reply by dong-lufei** [reply]
> You need to support md with something like https://github.com/unplugin/unplugin-vue-markdown

谢谢，您说的插件确实搞定我的问题了。我在里面还发现了另一个插件 vite-plugin-pages

**Reply by posva** [reply]
It should be the same config 🙂
